src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -805,8 +795,53 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
     }
 
     /// <summary>
-    /// Lp Normalizing estimator takes columns and normalizes them individually by rescaling them to unit norm.
+    /// Normalizes (scales) vectors in the input column to the unit norm. The type of norm that is used can be specified by the user.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of <xref:System.Single> |
+    /// | Output column data type | Vector of <xref:System.Single> |
+    ///
+    ///
+    /// The resulting <xref:Microsoft.ML.Transforms.LpNormNormalizingTransformer> normalizes vectors in the input column individually
+    /// by rescaling them to the unit norm.
+    ///
+    /// Let $x$ be the input vector, $n$ the size of the vector, $L(x)$ the norm function selected by the user.
+    /// Let $\mu(x) = \sum_i x_i / n$ be the mean of the values of vector $x$. The <xref:Microsoft.ML.Transforms.LpNormNormalizingTransformer>
+    /// performs the following operation on each input vector $x$:
+    ///
+    ///     $y = \frac{x - \mu(x)}{L(x)}$
+    ///
+    /// if the user specifies that the mean should be zero, or otherwise:
+    ///
+    ///     $y = \frac{x}{L(x)}$
+    ///
+    /// There are four types of norm that can be selected by the user to be applied on input vector $x$. They are defined as follows:
+    /// - <xref:Microsoft.ML.Transforms.Text.TextFeaturizingEstimator.NormFunction.L1>
```

Comment:
Should these be the Microsoft.ML.Transforms.LpNormNormalizingEstimatorBase.NormFunction values rather than the TextFeaturizing ones?

Reply:
Yes you are right! Thank you for the catch!!

---
In reply to: [277186012](https://github.com/dotnet/machinelearning/pull/3454#discussion_r277186012) [](ancestors = 277186012)